observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -24,6 +24,20 @@ The following table provides information about which features are available depe
 | 1.1+          | 4.18+              | ✔                     | ✔         | ✔                     | ✔          | ✔
 |===
 
+[id="cluster-observability-operator-release-notes-1-2-1_{context}"]
+== {coo-full} 1.2.1
+
+[id="cluster-observability-operator-1-2-1-new-features-enhancements_{context}"]
+=== New features and enhancements
+
+// Dev preview, so cannot be documented yet
+//* This release improves the functionality and ease of maintenance of Perses dashboards, through the reuse of variable selector and time range selection components. (link:https://issues.redhat.com/browse/COO-871[COO-871])
+
+// Dev preview, so cannot be documented yet
+//* Role-based access control (RBAC) has been added to Perses dashboards in {product-title}. (link:https://issues.redhat.com/browse/COO-870[COO-870])
+
+* This release adds support for OTEL formatted logs in the troubleshooting UI plugin (link:https://issues.redhat.com/browse/COO-365[COO-365]) and Korrel8r (link:https://issues.redhat.com/browse/COO-357[COO-357])
```

Comment:
Hello @gabriel-rh can you remove the updates below, @alanconway confirmed these will not be included in COO 1.2.1 (they will be pushed to next release COO 1.3.0). 

>* This release adds support for OTEL formatted logs in the troubleshooting UI plugin (link:https://issues.redhat.com/browse/COO-365[COO-365]) and Korrel8r (link:https://issues.redhat.com/browse/COO-357[COO-357])

[Slack Confirmation](https://redhat-internal.slack.com/archives/C0846BQHM3R/p1751481552991239)